pkg/workloads/console/controller.go

```diff
@@ -388,6 +433,10 @@ func (r *reconciler) updateStatus(job *batchv1.Job) error {
 		auditLog(r.logger, r.console, &duration, ConsoleEnded, "Console ended after reach expiration time")
 	}
 
+	if newStatus.Phase == workloadsv1alpha1.ConsoleDestroyed {
```

Comment:
The fact that we're always logging when the phase is destroyed, rather than only when we transition to destroyed, seems odd? Is this related to why we've got the predicates added?

Reply:
Ah yeah good catch! I'll change the conditional to only log when the existing state isn't already `workloadsv1alpha1.ConsoleDestroyed`.